manga_auto_translator/pipeline.py

```diff
@@ -1,12 +1,16 @@
 from typing import Sequence
 from manga_auto_translator.data_structure import Scan
 from manga_auto_translator.ocr import OcrStrategy
+from manga_auto_translator.translation import TranslationStrategy
 
 
 class TranslationPipeline:
-    def __init__(self, scans: Sequence[Scan], ocr_strategy: OcrStrategy) -> None:
+    def __init__(self, scans: Sequence[Scan], lang_from: str, lang_to: str, ocr_strategy: OcrStrategy, translation_strategy: TranslationStrategy) -> None:

```

Comment:
We can remove the 2 language arguments here if they are passed in the constructor arguments of the `TranslationStrategy`

Reply:
I second this